NuGet.config

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
It doesn't seem like we should need this file for accessing the published NuGets?

Reply:
Not for the released, but yes for the pre-released. We are adding a similar file to all our repos so it's easier to integrate x-repo changes before they become part of a release.